Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -170,9 +169,64 @@ public func verifySnapshot<Value, Format>(
   file: StaticString = #file,
   testName: String = #function,
   line: UInt = #line
+)
+-> String? {
+  verifySnapshot(
+    matching: try value(),
+    as: snapshotting,
+    named: name,
+    record: recording,
+    snapshotDirectory: snapshotDirectory,
+    timeout: timeout,
+    strategyIndex: 0,
+    file: file,
+    testName: testName,
+    line: line
   )
-  -> String? {
+}
+
+// MARK: - Private
+
+private func assertSnapshot<Value, Format>(
+  matching value: @autoclosure () throws -> Value,
+  as snapshotting: Snapshotting<Value, Format>,
+  named name: String? = nil,
+  record recording: Bool = false,
+  timeout: TimeInterval = 5,
+  strategyIndex: UInt,
```

Comment:
I didn't want to expose this parameter in public function. That's why I created a private function and call it from public one.